### DISCUSSION TITLE
Simplify the targeted client process for online marketing.

### DISCUSSION THREAD

**Discussion by JamesRho1987** [mettle/sendportal#245] [Ideas] [answered]
How can I do this so I can make a decent income and then jump onboard some new projects with working capital?

**Comment by Korrelay1985** [comment, marked as answer]
Hey JamesRho1987, great question about simplifying the targeted client process for online marketing. Have you checked out this article about "[marketing a service](https://www.brighttail.com/blog/what-is-marketing-as-a-service-and-4-killer-reasons-you-need-it/)"? It talks about identifying your target audience, creating a unique selling point, and utilizing social media to reach potential clients. It might give you some insights and ideas on how to streamline your approach. Good luck with your new projects and making that decent income!
